build.gradle

```diff
@@ -714,8 +714,8 @@ uiTestTargets.each { version ->
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
       // TODO (VirtusLab/ide-probe#195): remove the need to set these env variables
-      environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version
-      environment 'IDEPROBE_TEST_CASE', 'artifacts'
+      environment 'MY_CI_SCREENSHOTS_DIR', "/artifacts"
+      environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version + '/screenshots'
```

Comment:
Okay... so in the spirit of https://github.com/VirtusLab/ide-probe/pull/197#discussion_r904957348:

> The only official way to set the env var is by configuring probe screenshot path in the fixture, either through the .conf file or in code.

Can we get rid of `IDEPROBE_TEST_SUITE` here and just pass the path solely via `MY_CI_SCREENSHOTS_DIR`?

Reply:
yes :)